banks-interface/src/lib.rs

```diff
@@ -10,11 +10,19 @@ use solana_sdk::{
     transaction::{self, Transaction, TransactionError},
 };
 
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub enum TransactionConfirmationStatus {
+    Processed,
+    OptimisticallyConfirmed,
```

Comment:
I'm still stuck on the number of letters in "OptimisticallyConfirmed".   8 syllables! 
Just "Confirmed"?  "Settled" was also proposed in Discord.

Reply:
Oh, I like "Confirmed". This usage seems consistent with our documented definition of confirmed, and we can document the guarantees of "optimistic confirmation" elsewhere.

I'm not too clear on what "Settled" means in this context. The payment-processing meaning doesn't seem to apply.

Reply:
I took these new names for a spin in some `solana-test-validator` output updates that I'm working on, it felt nice:
```
$ solana-test-validator
Mint address: vines1vzrYbzLMRdu58ou5XTby4qAqVRLmqo36NKPTg
Ledger location: test-ledger
Log: test-ledger/validator.log
Identity: FXTGZfrgFeCgFXuFRQziNyZcnXftz56wfQLghfzqpK6j
Version: 1.6.0
JSON RPC URL: http://127.0.0.1:8899
JSON RPC PubSub Websocket: ws://127.0.0.1:8900/
Gossip Address: 127.0.0.1:1024
TPU Address: 127.0.0.1:53170
Faucet Address: 127.0.0.1:9900
⠤ 00:00:46 | Processed Slot: 93 | Confirmed Slot: 92 | Finalized Slot: 61 | Snapshot Slot: 0 | Transactions: 93 | ◎499.999540000
```


Last line: **Processed Slot: 93 | Confirmed Slot: 92 | Finalized Slot: 61**